### PR TITLE
various distro meta data maintenance

### PR DESCRIPTION
**Pull request by pohly**
Bumping release number and bblayer.conf version.

### CODE REVIEW

**Comment by pohly** [comment]
@kad okay to merge?

@Valtis all Beaglebone black devices were blacklisted during this PR's testing.

**Comment by Valtis** [comment]
Beaglebone power cutter required cable unplugging again but it should work now.

**Comment by pohly** [comment]
I don't think we should test this again; all we need is a +1 from someone.

File: meta-ostro/conf/distro/ostro.conf [hunk 1]
```diff
@@ -1,6 +1,12 @@
 DISTRO = "ostro"
 DISTRO_NAME = "Ostro OS"
-DISTRO_VERSION = "1.0~snapshot-${DATE}"
+# The version is bumped after each official release.
+# This implies that new releases go out with a version
+# string based on the predecessor plus the suffix, which
+# was deemed sufficient in the "[Ostro-dev] versioning Ostro OS"
```
Comment (gvancuts) [inline meta-ostro/conf/distro/ostro.conf]: How about adding a link to it, i.e.: https://lists.ostroproject.org/pipermail/ostro-dev/2016-May/000427.html ?

Comment (pohly) [inline meta-ostro/conf/distro/ostro.conf]: I find that to long and distracting. If someone is interested, they can find the discussion in their favorite location (local copy, via a web search engine, gmane if he had registered our list, etc.)  via the mail thread title.

**Comment by kad** [comment]
I would skip comments about mailing lists, but otherwise all ok.

**Comment by pohly** [comment]
Seems there's no consensus about how much the comments should say about the background discussions ;-} Opinions range from "provide direct link" to "not mention it at all". Let me make a judgment call here and stick to the middle ground, which is what I have now - mentioning that there was a discussion (because we do not want to start a discussion from scratch the next time someone has questions about that line) but without the URL.

I'll merge now because it might fix build issues.

**Pull request merged**